com.unity.render-pipelines.core/Tests/Editor/Volumes/VolumeComponentTests.cs

```diff
@@ -4,6 +4,7 @@
 using System.Linq;
 using System.Reflection;
 using UnityEditor.Rendering;
+using Type = System.Type;
```

Comment:
What other Type is there?

Reply:
No need, the autoinclude from visual added that to me while I was doing tests